src/sections/take-action/letter.tsx

```diff
@@ -0,0 +1,29 @@
+import type { FC } from 'react'
+import { documentToReactComponents } from '@contentful/rich-text-react-renderer'
+import type { Document } from '@contentful/rich-text-types'
+
+import { mainParagraphStyle } from '../../utils/renderer'
+import { HighlightedTitle } from '../../components'
+
+interface LetterProps {
+  openLetter: Document
+}
+
+const OpenLetter: FC<LetterProps> = ({ openLetter }) => {
+  return (
+    <div className="flex flex-col items-center">
+      <div>
+        <HighlightedTitle
+          title="An Open Letter"
+          size="large"
+          color="clementine"
+        />
+      </div>
+      <div className="space-y-4 px-60">
```

Comment:
```suggestion
      <div className="space-y-6 w-2/4">
```

This spacing should do the trick, after that it should be good to go!

Reply:
ok, updated this! but it's kind of a temporary fix, we should 100% create a more mobile-friendly option before quad day.